osu.Game/Online/Multiplayer/MultiplayerRoomUser.cs

```diff
@@ -16,6 +17,11 @@ public class MultiplayerRoomUser : IEquatable<MultiplayerRoomUser>
 
         public MultiplayerUserState State { get; set; } = MultiplayerUserState.Idle;
 
+        /// <summary>
+        /// The availability state of the beatmap, set to <see cref="DownloadState.LocallyAvailable"/> by default.
+        /// </summary>
+        public BeatmapAvailability BeatmapAvailability { get; set; } = new BeatmapAvailability(DownloadState.LocallyAvailable);
```

Comment:
will `Downloaded` be shown as a status to other users? not sure why `DownloadState` was reused here.

Reply:
`Downloaded` means `Importing`, really thinking about renaming that already.